shadows/framework/src/main/java/org/robolectric/shadows/ShadowBitmapFactory.java

```diff
@@ -225,6 +225,8 @@ protected static Bitmap decodeByteArray(
       return null;
     }
     Bitmap bitmap = create(desc, opts, image);
+    ShadowBitmap shadowBitmap = Shadow.extract(bitmap);
+    shadowBitmap.createdFromBytes = data;
```

Comment:
What about removing `shadowBitmap.createdFromBytes = data` from https://github.com/robolectric/robolectric/blob/master/shadows/framework/src/main/java/org/robolectric/shadows/ShadowBitmapFactory.java#L204-L205? Another `decodeByteArray` calls the variant you modified.
And could you help to add related tests for it?